template/nestJs/src/generateMigration.ts

```
@@ -0,0 +1,83 @@
+) => {
+  const queries = await connection.driver.createSchemaBuilder().log();
+
+  if (queries.upQueries.length === 0) {
+    Logger.warn('Database is up-to-date. ');
+    return;
+  }
+
+  const mapSqlToJs = (query: Query) => {
+    return (
+      'await queryRunner.query(`' +
+      query.query.replace(new RegExp('`', 'g'), '\\`') +
```

Biome (1.9.4)

[error] Use a regular expression literal instead of the RegExp constructor.

Regular expression literals avoid some escaping required in a string literal, and are easier to analyze statically.
Safe fix: Use a literal notation instead.


(lint/complexity/useRegexLiterals)